schemas/config.json

```diff
@@ -87,7 +87,6 @@
                             "title": "prompt",
                             "description": "A prompt of the current parameter",
                             "type": "string",
-                            "pattern": "[:?]$",
```

Comment:
@EmilySeville7cfg - notice I removed this.

Reply:
Not an issue. ;)